### DISCUSSION TITLE
Shutter tablet input

### DISCUSSION THREAD

**Discussion by Kuina-chan** [ppy/osu#22808] [Q&A] [answered]
### Type

Game behaviour

### Bug description

I updated to windows 11, hence I want a fresh install of lazer, go ahead and install from github, setting everything up and when trying to play, I found out that my tablet suffering from some hitches like in the clip. Tried to restart the game but no work.

### Screenshots or videos

https://user-images.githubusercontent.com/75318520/223352692-ea9fa8a0-df66-4663-b879-bc20ad8b663c.mp4

### Version

2023.305.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/10906483/database.log)
[input.log](https://github.com/ppy/osu/files/10906484/input.log)
[network.log](https://github.com/ppy/osu/files/10906485/network.log)
[performance.log](https://github.com/ppy/osu/files/10906486/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10906488/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10906489/updater.log)

**Comment by Axilotl17** [comment, marked as answer]
are you using Lazer's tablet driver?

Try turning off High Precision mouse in settings

**Reply by BunnyTub** [reply]
This works wonders for a TON of use cases.
But I think it's partially (mostly) Windows 11's fault.
It broke a lot of things during the transition from 10 to 11. Including games.

**Reply by Axilotl17** [reply]
Okay? i still think turning it off might help.

**Reply by Kuina-chan** [reply]
I used to ignore lazer's driver because it somewhat harder to determine the correct area. I use a strange ratio tbh. But now it seems that the fix for now. Thanks!!

**Comment by a0193143** [comment]
This also happend on Windows 10, could be temporary fixed either disable High Precision or use Lazer's driver.
